reference/library/src/webapp/editor/ckeditor.launch.js

```diff
@@ -155,87 +154,46 @@ sakai.editor.editors.ckeditor.launch = function(targetId, config, w, h) {
         templates: 'customtemplates'
     };
 
-    //NOTE: The height and width properties are handled discretely here.
-    //      The ultimate intent is that the caller-supplied config will simply
-    //      overlay the default config. The outstanding question is whether
-    //      some properties should disallow override (because of specific setup
-    //      here that we would not want duplicated throughout calling code) or
-    //      if their override would just be discouraged. We also probably want
-    //      some symbolic things like editorSize: 'small', where the supplied
-    //      values are interpreted and translated into dimensions, toolbar set,
-    //      and anything else relevant. This will allow editor indifference
-    //      on the part of tool code, requesting whatever editor be launched
-    //      with appropriate settings applied, rather than detecting the editor
-    //      and supplying specific values for the desired effect. This set of
-    //      "logical" configuration options is yet to be determined.
-    if (config) {
-        if (config.width) {
-            ckconfig.width = config.width;
-        } else if (w) {
-            ckconfig.width = w;
+    //To add extra plugins outside the plugins directory, add them here! (And in the variable)
+    (function() {
+        // SAK-30370 present a nice and simple editor without plugins to the user on a tiny screen.
+        if ($(window).width() < 768) {
```

Comment:
Should this number (768) match with the morpheus phone media query break point value (800)? 
```
$break-small-end: 	800px;
$break-medium-end: 	1030px;

$phone  :       "only screen and (max-width : #{ $break-small-end })";
$tablet :       "only screen and (max-width : #{ $break-medium-end })";
$desktop:       "only screen and (min-width : #{ $break-medium-end + 1 })";
```